README.md

```diff
@@ -1 +1,55 @@
-Review Past project
\ No newline at end of file
+# Review Past Project
+
+> This is a code review practice to test my code reviewing skills.
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+- Git
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+Clone the repository and change the directory to blog
+
+```
+  $ git clone git@github.com:Favourezeugwa/Review-past-project.git
+
+  $ cd Review-past-project
+```
+
+### Install
+
+Install all project dependencies by running the command below
+
+### Run the application
+
+
+### Visit the homepage
+
+
+### Check linter errors
+
+
+### Authors
+
+👤 **Favour Amarachi Ezeugwa**
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]().
```

Comment:
I think you can add a link to your repo's issue page, perhaps, someone may want to leave some suggestions. 💭

Reply:
- [ ]  Please, Instead of giving orders, make suggestions. *"I think it would be better to use semantic tags"* sounds nicer than *"Use semantic tags,"* doesn't it?  you can visit this link [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) for more on how to give a good Code Review.